client/web/src/site-admin/components/RepoMirrorInfo.tsx

```diff
@@ -18,7 +18,8 @@ export const RepoMirrorInfo: React.FunctionComponent<
                     <>Not yet synced from code host.</>
                 ) : (
                     <>
-                        Last synced <Timestamp date={mirrorInfo.updatedAt} />. Size:{' '}
+                        Last synced <Timestamp date={mirrorInfo.updatedAt} />. Next sync time:{' '}
+                        {mirrorInfo.due === null ? <>N/A</> : <Timestamp date={mirrorInfo.due} />}. Size:{' '}
```

Comment:
Should we skip `Next sync time: N/A` if there's no data?

```suggestion
                        Last synced <Timestamp date={mirrorInfo.updatedAt} />.
                        {mirrorInfo.due &&
                          <>Next sync time: <Timestamp date={mirrorInfo.due}</>
                        }
                        Size:{' '}
```

Reply:
I was contemplating this too, but I personally felt that it would confusing if we didn't show anything on some rows and the next sync time on others.

I'm cool going either way though.

Reply:
I do not feel strongly either. I just remember the feedback that this space is getting a little crowded. But on the other hand you're right - _Explicit is better than implicit._  as the zens of pythons say.

Reply:
Perhaps design would know?

I've got a pending issue as well to add cloning progress in the same space, so I think at that point, we'll have to propose something better than concatenated sentences anyways.

Reply:
I’m hoping the PR to remove the buttons from the rows helps clean the table up enough that we have more room to display more 😛

Reply:
let's ask @sourcegraph/design about that for sure!
I also cannot tell which option I like more...

Reply:
Under which circumstances do we not have a next-sync-at? If we know, can we express that in a sentence? I.e "sync disabled" instead of "next sync at: n/a".

Reply:
Yeah. I think "No update scheduled" sounds better.